x-pack/filebeat/input/cel/input.go

```diff
@@ -1161,7 +1191,7 @@ func cloneMap(dst, src mapstr.M) {
 // walkMap walks to all ends of the provided path in m and applies fn to the
 // final element of each walk. Nested arrays are not handled.
 func walkMap(m mapstr.M, path string, fn func(parent mapstr.M, key string)) {
-	key, rest, more := strings.Cut(path, ".")
+	key, rest, more := strings.Cut(path, ".") //nolint:typecheck // rest is used below
```

Comment:
Please don't add this. It is not correct, but rather an unfortunate interaction of a misuse of GOOS env vars by us in the CI and bad error reporting by the linter; it should have completely failed rather than giving this spurious and incorrect claim.